src/webserver/controllers/user_controller.go

```diff
@@ -54,7 +54,7 @@ func (u *User) Register(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if u.users.IsUsernameTaken(data.Username) {
-		u.log.Warnf("Username already taken, error %s", err.Error())
+		u.log.Warnf("Username already taken, error")
```

Comment:
Is the actual error message not relevant?

Reply:
There is no error message in this if branch. It was giving a nil pointer error